packages/flame/lib/src/cache/images.dart

```diff
@@ -7,12 +7,20 @@ import 'package:flutter/painting.dart';
 import 'package:flutter/services.dart';
 
 class Images {
-  Images({String prefix = 'assets/images/'}) {
+  Images({
+    String prefix = 'assets/images/',
+    AssetBundle? bundle,
+  }) {
     this.prefix = prefix;
+    this.bundle = bundle ?? Flame.bundle;
```

Comment:
Same here as the other one, and the prefix can be set directly in the constructor right?